coa-helm/templates/cbas-api.yaml

```diff
@@ -1,24 +1,29 @@
+{{- if .Values.cbas.coaEnabled }}
 {{ include "terra-batch-libchart.cbas-api-deploy" (list . "coa-helm.cbas-api-deploy") }}
+{{ end }}
 {{ define "coa-helm.cbas-api-deploy" }}
 # Overrides go here
 spec:
-  template:
-    metadata:
-      labels:
-    {{ include "coa-helm.labels" $ | nindent 8 }}
+template:
+  metadata:
+    labels:
+  {{ include "coa-helm.labels" $ | nindent 8 }}
 {{ end }}
 
 ---
 
-{{ include "terra-batch-libchart.cbas-api-service" (list . "coa-helm.cbas-api-service") }}
 {{ define "coa-helm.cbas-api-service" }}
+{{- if .Values.cbas.coaEnabled }}
```

Comment:
This (and others) still need to be fixed w.r.t. the above pattern

Reply:
Updated!